src/commands/ImportExportCommands.cpp

```diff
@@ -135,3 +138,22 @@ bool ExportCommand::Apply(const CommandContext & context)
    return false;
 }
 
+namespace {
+using namespace MenuTable;
+
+// Register menu items
+
+AttachedItem sAttachment{
+   wxT("Optional/Extra/Part2/Scriptables2"),
+   Items( wxT(""),
+      // Note that the PLUGIN_SYMBOL must have a space between words,
+      // whereas the short-form used here must not.
+      // (So if you did write "Compare Audio" for the PLUGIN_SYMBOL name, then
+      // you would have to use "CompareAudio" here.)
+      Command( wxT("Import2"), XXO("Import..."),
```

Comment:
The position of the "Import..." and "Export..." menu items moved from in between the "Help" and "Open Project...", now moved to the end of "Extra/Scriptables II" submenu.  Is this okay?